docs/products/opensearch/howto/sample-dataset.rst

```diff
@@ -0,0 +1,169 @@
+Sample dataset
+--------------
+Here you will find different sample datasets for OpenSearch and how to interact with them.
+
+Before exploring the available datasets, we need an OpenSearch server to load the data on. A quick and easy way to get a server is through `Aiven for OpenSearch <https://aiven.io>`_. If don't have an Aiven account yet, `sign up <https://console.aiven.io/signup?utm_source=github&amp;utm_medium=organic&amp;utm_campaign=devportal&amp;utm_content=repo>`_ and enjoy our free trial!
+
+After logging in the Aiven Console, chose the OpenSearch version, select the cloud and plan of your choice, give the service a name and hit the "Create Service" button. In a couple of minutes, you will have an OpenSearch server up and running.
+
+Epicurious recipes
+==================
```

Comment:
Let's engage the user a little here, it's important to be factual but I think "a great way to try the service is to play with it, and who doesn't like to play with their food?" type of approach to introduce some fun-but-harmless sample data to get people going would be good before diving in.

Reply:
updated